src/biokbase/narrative/viewers.py

```diff
@@ -17,22 +29,36 @@ def view_as_clustergrammer(ws_ref, col_categories=(), row_categories=(), normali
         'row' or 'column' axis
     :return:
     """
-    assert isinstance(col_categories, (tuple, set, list))
-    assert isinstance(row_categories, (tuple, set, list))
-    assert normalize_on in {None, "row", "column"}
+    errors = []
+    if not isinstance(col_categories, tuple | set | list):
+        errors.append("col_categories must be a tuple, set, or list")
+    if not isinstance(row_categories, tuple | set | list):
+        errors.append("row_categories must be a tuple, set, or list")
+    if normalize_on not in {None, "row", "column"}:
+        errors.append("normalize_on must be 'row', 'column' or None")
+
+    if errors:
+        error_msg = "Error with clustergrammer arguments:\n" + "\n".join(errors)
+        raise AssertionError(error_msg)
 
     generic_df = get_df(ws_ref, col_categories, row_categories, True)
 
     net = Network(clustergrammer_widget)
-    net.df_to_dat({"mat": generic_df})
+    # load pandas dataframe
+    net.load_df(generic_df)
     if normalize_on:
         net.normalize(axis=normalize_on)
-    net.cluster(enrichrgram=False)
+    net.cluster()
```

Comment:
enrichrgram either defaults to False or is no longer a valid value - I don't remember which.